forge-gui/res/cardsfolder/upcoming/bank_job.txt

```diff
@@ -0,0 +1,13 @@
+Name:Bank Job
+ManaCost:1 B R
+Types:Enchantment
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | Execute$ TrigExile | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of your upkeep, exile the bottom creature card of your library. You may cast that card this turn. At the beginning of the next end step, if that card is still exiled, put it into your graveyard and create a Treasure token.
+SVar:TrigExile:DB$ ChangeZoneAll | ChangeType$ Card.YouOwn+BottomLibrary_Creature | Origin$ Library | Destination$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ MayCast | RememberObjects$ Remembered | ExileOnMoved$ Exile | SubAbility$ DBDelayedTrigger
+SVar:MayCast:Mode$ Continuous | Affected$ Card.IsRemembered+nonLand | MayPlay$ True | EffectZone$ Command | AffectedZone$ Exile | Description$ You may cast that card this turn.
+SVar:DBDelayedTrigger:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigGrave | IsPresent$ Card.IsRemembered | PresentZone$ Exile | RememberObjects$ Remembered | SubAbility$ DBCleanup | TriggerDescription$ At the beginning of the next end step, if that card is still exiled, put it into your graveyard and create a Treasure token.
```

Comment:
I need to check, I think this may need to be `IsPresent$ Card.IsTriggerRemembered`
And line 10 `Defined$ DelayTriggerRememberedLKI`

Reply:
I have no idea why, but this is fine as it is... whereas Grinning Totem definitely needs the above or it does not work. 🤷‍♂️

Reply:
Ugh, thanks a lot for the heads up

Please always use `DelayTriggerRememberedLKI`
(it only works right now because it creates LKI for the trigger host but I'm trying to fix that)